stable/cert-manager/templates/00-namespace.yaml

```diff
@@ -3,4 +3,7 @@ apiVersion: v1
 kind: Namespace
 metadata:
   name: {{ .Release.Namespace | quote }}
+  labels:
+    name: {{ .Release.Namespace | quote }}
+    certmanager.k8s.io/disable-validation: "true"
```

Comment:
This file is only created when generating static deployment manifests, and the updates here are to keep it in line with default Helm 2.10 behaviour (and the second one is to help with the webhook deployment, and is effectively a no-op as we also match on the 'name' label)